src/main/java/com/google/api/codegen/go/GoDiscoveryContext.java

```diff
@@ -64,17 +64,12 @@ public String typeDefaultValue(Type type, Field field) {
       return DEFAULT_VALUES.get(field.getKind());
     }
     if (field.getKind() == Field.Kind.TYPE_STRING || field.getKind() == Field.Kind.TYPE_ENUM) {
-      return getDefaultString(type, field);
+      return getDefaultString(type, field).getDeclare();
```

Comment:
nit: Is getDeclare actually a getDefine? You're defining the value, not declaring the type, IIC.

(We can fix the names later if need. I'm probably misunderstanding.)